boofuzz/__init__.py

```diff
@@ -561,7 +576,7 @@ def s_group(name, values, default_value=None):
     """
 
     group = primitives.Group(name, values, default_value)
-    blocks.CURRENT.push(group)
+    blocks.CURRENT.push(FuzzableWrapper(fuzz_object=group, name=name, fuzzable=False, default_value=default_value))
```

Comment:
The `default_value` passed in `Group()` can be removed too, since the `Group._value` and `Group._original_value` are not used anymore.